src/common/snippets/src/lowered/loop_manager.cpp

```diff
@@ -15,8 +15,17 @@
 namespace ov {
 namespace snippets {
 namespace lowered {
+using LoopManager = LinearIR::LoopManager;
+using LoopPort = LoopManager::LoopPort;
+using LoopInfo = LoopManager::LoopInfo;
+
+std::shared_ptr<LoopPort> LoopPort::clone_for_new_expr(const ExpressionPtr& new_expr) const {
```

Comment:
Now we can clone `LoopPort` 🎆  May I ask you to update the WA [there](https://github.com/openvinotoolkit/openvino/blob/3403e6c02873bb0fa9f0cf410d24bbb79991e93c/src/common/snippets/src/lowered/loop_manager.cpp#L323) please?

Reply:
I tried, but please double-check. I'm not 100% confident that I got it right 😅

Reply:
You did exactly what I meant. Thanks!

Reply:
@IvanNovoselov  @a-sidorova Actually we can't just use `clone_for_new_expr` instead of old logic because 
new expression ports in `update_loop_port` might have different `port_idx` besides expression. Let's look at a specific example. E.g. we use `update_loops_port` when we insert Load operation on 1st Add's input: in this case new ExpressionPort has idx=0 whereas Add's ExpressionPort has idx=1.

So we can return old logic or introduce a new API for LoopPort: `clone_with_new_expr_port`

Reply:
@v-Golubev Oh, I see your point. Great catch! Thanks a lot!

@IvanNovoselov I'm sorry that I confused you. Could you please revert this change? It's invalid since we call `clone_for_new_expr` with new expr that isn't the same as previous expr (they may have different input count, output count). I think that in this case we should add `clone_with_new_expr_port`. You may just revert this change and don't introduce new API with `clone_with_new_expr_port`. It's just small suggestion.

Reply:
All right, I reverted the changes